pkg/await/extensions_ingress_test.go

```diff
@@ -78,6 +78,20 @@ func Test_Extensions_Ingress(t *testing.T) {
 					"Ingress has at least one rule that does not target any Service. " +
 						"Field '.spec.rules[].http.paths[].backend.serviceName' may not match any active Service"}},
 		},
+		{
+			description:  "Should succeed for Ingress with an unspecified path.",
+			ingressInput: ingressInput,
+			do: func(ingresses, services, endpoints chan watch.Event, settled chan struct{}, timeout chan time.Time) {
+				// API server passes initialized ingress back.
+				ingresses <- watchAddedEvent(initializedIngressUnspecifiedPath("default", "foo", "foo-4setj4y6"))
+				endpoints <- watchAddedEvent(initializedEndpoint("default", "foo-4setj4y6"))
+
+				settled <- struct{}{}
```

Comment:
nit: can we rename this to `endpointsSettled` or similar to be more explicit?

Reply:
Yeah, I'm planning to do a cleanup pass on these tests soon, so I'll get it then.